packaging/hipify-clang.txt

```diff
@@ -12,9 +12,9 @@ set(CPACK_PACKAGE_VENDOR "Advanced Micro Devices, Inc.")
 set(CPACK_PACKAGE_VERSION_MAJOR @LLVM_VERSION_MAJOR@)
 set(CPACK_PACKAGE_VERSION_MINOR @LLVM_VERSION_MINOR@)
 set(CPACK_PACKAGE_VERSION_PATCH @LLVM_VERSION_PATCH@)
-#TODO : Change mail id to standard <package.suppport@amd.com>
-set(CPACK_PACKAGE_CONTACT "Mankov Evgeny <Evgeniy.Mankov@amd.com>")
-set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "hipify-clang: a clang-based tool to translate CUDA source code into portable HIP C++ automatically")
+set(CPACK_PACKAGE_CONTACT "ROCm Compiler Support <rocm.compiler.support@amd.com>")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Hipify CUDA source.")
+set(CPACK_PACKAGE_DESCRIPTION "This tool translates CUDA source code into portable HIP C++, automatically")
```

Comment:
I'd suggest the following:
"The tool translates CUDA source code into portable HIP C++ automatically"

Reply:
ok